editor/index.html

```diff
@@ -38,6 +38,7 @@
 		<script src="../examples/js/loaders/TDSLoader.js"></script>
 		<script src="../examples/js/loaders/VRMLLoader.js"></script>
 		<script src="../examples/js/loaders/VTKLoader.js"></script>
+		<script src="../examples/js/loaders/LegacyJSONLoader.js"></script>
```

Comment:
This is not the correct path. It should be:
```js
<script src="../examples/js/loaders/deprecated/LegacyJSONLoader.js"></script>
```

Reply:
Oh yes!! My bad. Thank you :)